who_is_active.sql

```diff
@@ -150,120 +153,117 @@ OUTPUT COLUMNS
 --------------
 Formatted/Non:	[session_id] [smallint] NOT NULL
 	Session ID (a.k.a. SPID)
-

```

Comment:
Can you put back the various removed line breaks? They're important for help output and general readability.

Reply:
np, I will clean this all up.